tests-example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-all.zip
+distributionUrl=https://services.gradle.org/distributions/gradle-8.8-bin.zip
```

Comment:
Why did you downgrade gradle version? It should be aligned with current react-native version.